testkit/Dockerfile

```diff
@@ -49,6 +49,8 @@ RUN update-ca-certificates --verbose
 # Add Deno
 RUN curl -fsSL https://deno.land/x/install/install.sh | sh
 RUN mv /root/.deno/bin/deno /usr/bin/
+# Using System CA in Deno
+ENV DENO_TLS_CA_STORE=system
```

Comment:
Don't forget to get that documented so that users know how to TLS when using the DenoJS driver.